app/code/core/Mage/Core/Helper/Data.php

```diff
@@ -153,31 +153,50 @@ public function formatPrice($price, $includeContainer = true)
     /**
      * Format date using current locale options and time zone.
      *
-     * @param   string|Zend_Date|null $date
+     * @param   string|Zend_Date|int|null $date If empty, return current local datetime.
      * @param   string              $format   See Mage_Core_Model_Locale::FORMAT_TYPE_* constants
      * @param   bool                $showTime Whether to include time
      * @return  string
      */
     public function formatDate($date = null, $format = Mage_Core_Model_Locale::FORMAT_TYPE_SHORT, $showTime = false)
     {
+        return $this->formatTimezoneDate($date, $format, $showTime);
```

Comment:
For v20 compatibility, maybe something like this:
```suggestion
        return func_num_args() === 4
             ? $this->formatTimezoneDate($date, $format, $showTime, func_get_args()[3]);
             : $this->formatTimezoneDate($date, $format, $showTime);
```

Reply:
Would say no. v20 change was added four month ago in 08/2022 .... if someone wrote code that uses 4th param, he/she should be able to change that line. _imho_